src/components/partnership/partnership.service.ts

```diff
@@ -168,8 +168,7 @@ export class PartnershipService {
     { organizationId, projectId, ...input }: CreatePartnership,
     session: ISession
   ): Promise<Partnership> {
-    const id = generate();
-    const createdAt = DateTime.local();
+    const createdAt = DateTime.local().toString();
```

Comment:
There's no need to convert DateTime's to strings like this. We automatically convert them to Neo4J DateTime objects. Doing this string conversion actually prevents this.